### DISCUSSION TITLE
Image name

### DISCUSSION THREAD

**Discussion by vishalrijhwani** [eclipse/jkube#2031] [General]
Whenever i create kubernetes.yaml file through jkube through gradle image name is spring-boot. I need to give my own image name to it
![image](https://user-images.githubusercontent.com/82660507/215543923-36df53fc-ffe0-4df3-933d-74b04f1e83e0.png)

**Comment by rohanKanojia** [comment]
Are you using Kubernetes Gradle Plugin :detective: ?

The field you're pointing to is the container name, not the image name. Do you want to change name of container or name of the image (the one specified in `image:`?

**Comment by rohanKanojia** [comment]
The container name usually maps to Image Alias by default. If you provide property `jkube.generator.alias=custom-container-name`, it should work

**Reply by rohanKanojia** [reply]
@vishalrijhwani : Did my suggestion of using `gradle k8sResource -Pjkube.generator.alias=custom-name` work for your use case?

**Comment by vishalrijhwani** [comment]
Yes it works

Thanks and regards
Vishal

On Tuesday, January 31, 2023, Rohan Kumar ***@***.***> wrote:

> @vishalrijhwani <https://github.com/vishalrijhwani> : Did my suggestion
> of using gradle k8sResource -Pjkube.generator.alias=custom-name work for
> your use case?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse/jkube/discussions/2031#discussioncomment-4830660>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATWUZGZ6O2PJVLACBMGQ7CLWVEPYVANCNFSM6AAAAAAULMU47Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Vishal Rijhwani

**Reply by rohanKanojia** [reply]
If it worked, Could you please mark the answer as accepted and close the discussion?

**Comment by vishalrijhwani** [comment]
It works. It’s closed
